app/controllers/miq_policy_controller/policies.rb

```diff
@@ -31,7 +31,7 @@ def policy_edit_reset
   end
 
   def policy_edit_save
-    assert_privileges("policy_#{@policy.id ? "edit" : "new"}")
+    assert_privileges("miq_policy_#{@policy.id ? "edit" : "new"}")
```

Comment:
These are the ones that prevent non-admin users to edit/reset/save policies in the UI currently.